packages/radios/src/views/Label.example.md

```diff
@@ -1,38 +1,40 @@
 ### States
 
 ```jsx
-<Grid>
-  <Row>
-    <Col md={4}>
-      <Radio>
-        <Label regular>Regular Label</Label>
-      </Radio>
-    </Col>
-    <Col md={4}>
-      <Radio checked onChange={() => console.log('checked value changed')}>
-        <Label>Checked Label</Label>
-      </Radio>
-    </Col>
-    <Col md={4}>
-      <Radio>
-        <Label hidden>Hidden Label</Label>
-      </Radio>
-    </Col>
-    <Col md={4}>
-      <Radio>
-        <Label hovered>Hovered Label</Label>
-      </Radio>
-    </Col>
-    <Col md={4}>
-      <Radio>
-        <Label focused>Focused Label</Label>
-      </Radio>
-    </Col>
-    <Col md={4}>
-      <Radio disabled>
-        <Label>Disabled Label</Label>
-      </Radio>
-    </Col>
-  </Row>
-</Grid>
+<div role="group" aria-label="State Usage Example">
```

Comment:
Can these attributes apply to `<Grid>`? In other words, make `role="group"` automatic with the component and apply `aria-label` manually.

Reply:
These are no added to the `<Grid>`